code-of-conduct/enforcement-manual.md

```diff
@@ -0,0 +1,98 @@
+- whether there is a threat to anyone's physical safety
+- what happened
+- whether this event constitutes a Code of Conduct violation
+- who, if anyone, was the bad actor
+
+This information will be collected in writing.
+The Code of Conduct committee will provide a written summary of the information surrounding the incident.
+All participants will be anonymized in the summary report, referred to as "Community Member 1", "Member project 1", or "Board member 1" and so on.
+The "de-anonymizing key" will be kept in a separate file and only accessed to link repeated reports against the same person over time.
+
+The Code of Conduct committee will aim to have a resolution agreed upon within one week.
+If a resolution can't be determined in that time, a member of the Code of Conduct committee will respond to the reporter(s) with an updated and projected timeline for resolution.
+
+Once the committee has determined its resolution, the original reporter will be contacted to let them know what action (if any) will be taken. 
+The committee will take into account feedback from the reporter on the appropriateness of its response but may decide not to act on that feedback.
+
+The Chair of the Code of Conduct Committee will write up a transparency report for incidents reported through the incident report form or other channels. 
+Names of the reporter and any persons involved in the incident will not be included unless the resolution results in a termed suspension. 
+The CoC committee may choose to make a public report of the incident while maintaining anonymity of those involved.
+
+## 4.3 Following up with the Reportee
+
+When following up with the reportee, the committee representatives will:
+- Explain that an incident was reported that involves the reportee.
+- In this explanation, the focus will be on the impact of their behaviour, not their intent.
+- Reiterate the Code of Conduct and that their behaviour was deemed inappropriate.
+- Provide concrete examples of how they can improve their behaviour.
+- Give them the opportunity to state their view of the incident.
+- Remind them of the consequences of their behaviour, or future consequences if the behaviour is repeated.
+- Explain the possible resolutions that may be enforced should the committee determine there is a breach.
+- Conduct a final meeting to ensure that the reportee understands the resolution and has an opportunity to clarify concerns. In some cases, this final communication could be carried over email at the discretion of the committee.
+
+## 4.4 Resolutions
+
+The Code of Conduct committee will seek to agree on a resolution by consensus of all members investigating the report in question.
+If the committee cannot reach consensus and deadlocks for over a week, the appointed members from the OBF Board will break the tie.
+If these members are unable to take part in the discussion due to a conflict of interest, the President of the OBF Board will make the decision.
```

Comment:
I don't believe this needs adding to the main bylaws under the responsibilities of the officer roles. i.e. No change needed.

Reply:
I would agree.